src/App.js

```diff
@@ -1,6 +1,11 @@
 import React, { Component } from 'react';
 
 class App extends Component {
+  onGetStarted = () => {
+    const url = 'https://docs.google.com/forms/d/1TraoygcJay39cLiBQM-ROlToy7xIgcdx_VvCQuE1lJw/edit#responses';
```

Comment:
First of all, great catch!
Second, unless there is a specific reason or behavior you want to obtain I would not use this approach.
There is far way easier way to obtain this via simple CSS, working with `padding` and such targeting the `a` as a block element.

Beside that, CSS comes with the advantage of setting up a default style for this kind of action – so, in future, all buttons would look the same and have ideally one single point from where style and behavior is defined.

Reply:
It's totally fine to commit on the PR again, sure!
Great job and also great reaction time 😄